src/java/picard/sam/MergeSamFiles.java

```diff
@@ -52,13 +52,30 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Merges multiple SAM/BAM files into one file.",
-        usageShort = "Merges multiple SAM or BAM files into one file",
+        usage = MergeSamFiles.USAGE_SUMMARY + MergeSamFiles.USAGE_DETAILS,
+        usageShort = MergeSamFiles.USAGE_SUMMARY,
         programGroup = SamOrBam.class
 )
 public class MergeSamFiles extends CommandLineProgram {
     private static final Log log = Log.getInstance(MergeSamFiles.class);
 
+    static final String USAGE_SUMMARY = "Merges multiple \".sam\" and/or \".bam\" files into one file.  ";
+    static final String USAGE_DETAILS = "This tool is used for combining \".sam\" and/or \".bam\" files from different runs or read groups.  " +
+            "<br /><br />To prevent errors in downstream processing, it is critical to identify/label read groups appropriately." +
+            "However, if different samples contain identical read group IDs (collisions) this tool makes modifications " +
```

Comment:
"However, " -> "Note that "
Remove "(collisions)"
"this tool ..." -> ", this tool will prevent collisions by modifying the read group IDs to be unique."